src/main/java/com/restaurante/controllers/CompanyController.java

```diff
@@ -14,22 +20,28 @@ public class CompanyController {
     private final CompanyServices companyServices;
 
     @PostMapping
-    public ResponseEntity<Object> createCompany(@RequestBody CompanyDto companyDto) {
-        return companyServices.createCompany(companyDto);
+    public ResponseEntity<Object> createCompany(@RequestBody CompanyDto companyDto)
+            throws CompanyExistException, InternalServerErrorException {
+        companyServices.createCompany(companyDto);
+        return ResponseEntity.status(HttpStatus.CREATED).body("Company created.");
     }
 
     @GetMapping
     public ResponseEntity<Object> listCompany(){
-        return companyServices.listCompany();
+        List<CompanyDto> lista = companyServices.listCompany();
+        return ResponseEntity.status(HttpStatus.OK).body(lista);
     }
 
     @PatchMapping
-    public ResponseEntity<Object> updateCompany(@RequestBody CompanyDto companyDto){
-        return companyServices.updateCompany(companyDto);
+    public ResponseEntity<Object> updateCompany(@RequestBody CompanyDto companyDto) {
+        companyServices.updateCompany(companyDto);
+        return ResponseEntity.status(HttpStatus.OK).body("The company has been updated.");
+
     }
 
     @DeleteMapping
-    public ResponseEntity<Object> deleteCompany(){
-        return companyServices.deleteCompany();
+    public ResponseEntity<Object> deleteCompany() {
+        companyServices.deleteCompany();
+        return ResponseEntity.status(HttpStatus.OK).body("Company is deleted.");
```

Comment:
Geralmente no **delete** _method_, retornamos o _Status Code_ **204**, que é significa **No Content**, e ele é um **Void**

Exemplo abaixo.

```suggestion
        return ResponseEntity.noContent().build();
```